container/podman/podman.go

```diff
@@ -108,7 +108,15 @@ func Status() (v1.DockerStatus, error) {
 		return v1.DockerStatus{}, err
 	}
 
-	return docker.StatusFromDockerInfo(*podmanInfo)
+	status, err := docker.StatusFromDockerInfo(*podmanInfo);
+	if err != nil {
+		return v1.DockerStatus{}, err
+	}
+
+	status.Version, _ = VersionString()
+	status.APIVersion, _ = APIVersionString()
```

Comment:
Can you log these errors, please?

Reply:
I was ignoring them, because both fields are set to "Unknown" by  `docker.StatusFromDockerInfo()` above.
Similarly both functions also return "Unknown" in case of an error.
I've pushed https://github.com/google/cadvisor/pull/3552/commits/31c860f4a30cf4ab72218a0eefd3c5be3ef1e60b with error handling to make this behavor more explicit.
Please have a look. :smiley: